### DISCUSSION TITLE
Return items count to groups in the File Explorer

### DISCUSSION THREAD

**Discussion by teslasystemsx** [valinet/ExplorerPatcher#3306] [Ideas]
In all previous Windows versions when file grouping was enabled in the File Explorer, it showed items count in the brackets for each group. Can you return this feature for Windows 11? See screenshot.

<img width="327" alt="Count" src="https://github.com/valinet/ExplorerPatcher/assets/140281857/a3e5ca69-15b6-42fe-a935-9af0f1914db2">

**Comment by pyrates999** [comment]
This is beyond what EP is for, sorry.
